vmdb/app/assets/javascripts/miq_dhtmlxgrid.js

```diff
@@ -63,20 +68,24 @@ function miqCheck_AE_All(button_div,gridname) {
     crows = ns_list_grid.getCheckedRows(0);
   } else if (typeof ns_grid != "undefined" && gridname == "ns_grid") {
     state = $('#Toggle2').prop('checked');
-    ns_grid.checkAll(state ? true:false)
+    ns_grid.checkAll(state ? true : false)
     crows = ns_grid.getCheckedRows(0);
   } else if (typeof instance_grid != "undefined" && gridname == "instance_grid") {
     state = $('#Toggle3').prop('checked');
-    instance_grid.checkAll(state ? true:false)
+    instance_grid.checkAll(state ? true : false)
     crows = instance_grid.getCheckedRows(0);
   } else if (typeof class_methods_grid != "undefined" && gridname == "class_methods_grid") {
     state = $('#Toggle4').prop('checked');
-    class_methods_grid.checkAll(state ? true:false)
+    class_methods_grid.checkAll(state ? true : false)
     crows = class_methods_grid.getCheckedRows(0);
   }
-  if (miqDomElementExists('miq_grid_checks')) $('#miq_grid_checks').val(crows)
-  if (miqDomElementExists('miq_grid_checks2')) $('#miq_grid_checks2').val(crows)
-  count = crows == "" ? 0:crows.split(",").length;
+  if (miqDomElementExists('miq_grid_checks')) {
+    $('#miq_grid_checks').val(crows)
```

Comment:
Missing semicolons are added in another PR: #3021. Here are only cosmetic changes.